400-claims.md

```diff
@@ -0,0 +1,197 @@
+
+## Concepts of Package Management
+
+A _package_ is a discrete data chunk that can be moved from location to location, and can be unpacked and installed onto a system. Typically, a package contains an application or application description. All package managers provide some explicit definition of a package and a package format.
+
+When a package is installed, the contents of a package are extracted and placed into the appropriate spaces on the target system, thus becoming an _installation_ (or _instance_) of the package.
+
+There are three core feature categories of a package manager system:
+
+- It can _install_ packages (initially put something onto a system)
+- It can _query_ installations (to see what is installed)
+- It can _upgrade_ and _delete_ packages (in other words, it can perform additional mutations on an existing installation)
+
+Package managers provide a wealth of other features, but the above are standard across all package managers. (For example, most package managers also provide a way to query what packages are available for installation.)
+
+This specification explains how CNAB records are generated such that continuity can be established across applications. In other words, this describes how CNAB bundles can be treated analogously to traditional packages.
+
+## Managing State
+
+Fundamentally, package managers provide a state management layer to keep records of what was installed. For example, [homebrew](http://homebrew.sh), a popular macOS package manager, stores records for all installed software in `/usr/local/Cellar`. Helm, the package manager for Kubernetes, stores state records in Kubernetes ConfigMaps located in the system namespace. The Debian Apt system stores state in `/var/run`. In all of these cases, the stored state allows the package managing system to be able to answer (quickly) the question of whether a given package is installed.
+
+```console
+$ brew info cscope
+cscope: stable 15.8b (bottled)
+Tool for browsing source code
+https://cscope.sourceforge.io/
+/usr/local/Cellar/cscope/15.8b (10 files, 714.2KB) *
+  Poured from bottle on 2017-05-15 at 09:24:58
+From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/cscope.rb
+```
+
+The CNAB Claims specification does not define where or how records are stored, nor how these records MAY be used by CNAB tooling. However, this specification describes how a CNAB-based system MUST emit the record to an invocation image, and provides some guidance on maintaining integrity of the system.
+
+This is done so that implementors can standardize on a way of relating a release claim (the record of a release) to release operations like `install`, `upgrade`, or `uninstall`. This, in turn, is necessary if CNAB bundles are expected to be executable by different implementations.
+
+### Anatomy of a Claim
+
+While implementors are not REQUIRED to implement claims, this is that standard format for claims-supporting systems.
```

Comment:
should `this is that` read `this is the`?

Reply:
Fixed